lms/djangoapps/experiments/utils.py

```diff
@@ -27,6 +25,9 @@
 from student.models import CourseEnrollment
 from xmodule.partitions.partitions_service import get_all_partitions_for_course, get_user_partition_groups
 
+# Import this for backwards compatibility (so that anyone importing this function from here doesn't break)
+from .stable_bucketing import stable_bucketing_hash_group
```

Comment:
Since the experiments code is intended to be temporary, is this necessary?

Reply:
I'm mostly worried about any existing code that might be importing this from the old location that I might have missed.